src/main.ts

```
@@ -9,6 +9,7 @@ async function bootstrap() {
   forbidNonWhitelisted:true,
   transform:true
   }))
+  app.enableCors()
```

ESLint

[error] Insert `;`

(prettier/prettier)